### PR TITLE
Add torchtune to docker image packages

### PR DESCRIPTION
**Pull request by psbang**

### CODE REVIEW

File: tests/test_torchtune.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import unittest
+
+import subprocess
+
+class TestAlbumentations(unittest.TestCase):
```
Comment (rosbo) [inline tests/test_torchtune.py:5]: rename test class to `TestTorchtune`.

**Review by rosbo — COMMENTED** [review]
Thanks. Let's add @calderjo given he is the owner now.

File: tests/test_torchtune.py (hunk 1 above)
Comment (psbang) [reply tests/test_torchtune.py]: Wow, this is why we get PRs reviewed 🤦🏾‍♂️

**Review by calderjo — APPROVED** [review]
LGTM thanks!

**Pull request merged**